client/me/purchases/billing-history/utils.tsx

```diff
@@ -229,6 +229,31 @@ function renderSpaceAddOnquantitySummary(
 	} );
 }
 
+export function renderTransactionVolumeSummary(
```

Comment:
Nit: If the function is supposed to render only domain products, maybe it could be renamed to something more specific?

Reply:
Makes sense, will update, thanks!

Reply:
This was updated.